static/css/things.css

```diff
@@ -22,7 +22,7 @@
 #add-button {
   bottom: 2rem;
   right: 2rem;
-  background-image: url('../optimized-images/add.svg');
+  background: no-repeat center/100% url('../optimized-images/add.svg');
```

Comment:
This is strange, is it working around a bug in Safari? If so, may be worth leaving a comment.

Reply:
Yeah, there's some WebKit issue around this. I wish I had an exact pointer for a bug.